src/neptune_mlflow_plugin/impl/__init__.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+__all__ = ["create_neptune_tracking_uri"]
+
+from typing import Optional
+
+PLUGIN_SCHEME = "neptune-plugin"
+
+
+def create_neptune_tracking_uri(
+    *,
+    api_token: Optional[str] = None,
+    project: Optional[str] = None,
+    mode: str = "async",
+    name: str = "Untitled",
+    description: str = "",
+    capture_stdout: bool = True,
+    capture_stderr: bool = True,
+    capture_hardware_metrics: bool = True,
+    fail_on_exception: bool = True,
+    monitoring_namespace: str = "monitoring",
+    flush_period: float = 5,
+    capture_traceback: bool = True,
+) -> str:
+    return (
```

Comment:
Think of using Python internal URL building mechanism like here: https://stackoverflow.com/questions/15799696/how-to-build-urls-in-python-with-the-standard-library . It should be easier to extract some information further. We can use project as a "host" (or a placeholder if wasn't provided etc.).